let x = 32;

try {
    x = y + 1
    print(x)
} catch {
    print(error)
}